### DISCUSSION TITLE
Alternatif siteler için de bir çalışma olacak mı?

### DISCUSSION THREAD

**Discussion by gokhanmtlu** [KebabLord/turkanime-indirici#74] [Genel]
Programı başarılı buldum ve alternatif 2-3 site için de çalışma yapılırsa çok beğenileceğini düşünüyorum. Örneğin diziwatch(anime için) veya cizgivedizi(çizgi film için)

**Comment by KebabLord** [comment]
Bugün itibariyle, teknik anlamda TürkAnime; Türkiye'nin en fazla içeriğine, fansub çeşitliliğine ve alternatif player'larına sahip platform. Ayrıca 14 yıldır da sorunsuz çalıştığından, stabillik konusunda rakibi yok. Yani zaten en iyi alternatif gibi birşey.

Yine de, alternatif bir site, script'i hantallaştırmakta olan selenium bağımlılığından kurtaracaksa entegrasyonu kesinlikle düşünülebilir.

Ama bu bağlamda entegre edilebilecek alternatif site kesinlikle milletin emeğini çalan diziwatch gibi platform olmazdı.
Muhtemelen AnimeciX gibi kaliteli bir platformu seçerdim.

TürkAnime TV'ye meteor çarpmazsa yakın zamanda alternatif bir site ekleme fikrim yok. İşlerim yoğun, pek vaktim kalmıyor

Son 5 yıldır her aralık/ocak ayında yılda bir defaya mahsus olmak suretiyle yeni bir sürüm yayınlıyorum.
Yine de bu seneki yeni sürüm için aklımda olan asıl yeni özellik script'e gui getirmek olacak sanırım.

**Reply by gokhanmtlu** [reply]
Tekrardan elinize sağlık ve cevap için teşekkürler. Evet animeler için yeterli fakat çizgi film de sevenler için cizgivedizi güzel bir alternatif olabilirdi. Tekrardan cevap için teşekkürler, iyi günler dilerim.

**Reply by KebabLord** [reply]
asıl ben bu feedback için çok teşekkür ederim, script'i birilerinin kullandığını görmek beni çok mutlu ediyor. 

Bu arada cizgivedizi'yi inceledim, bu tür bir site için script yazmak kulağa oldukça kolay geliyor. Cloudflare yok, reklam yok, videolar direk sibnet. Daha da önemlisi içerik listesi, özet listesi, logo listesi csv olarak saklanıyor yazılımla ayrıştırması kolay.
https://www.cizgivedizi.com/film/isim.txt
https://www.cizgivedizi.com/film/logo.txt
https://www.cizgivedizi.com/film/ozet.txt

Benim script selenium kullandığından çizgivedizi'yi buna entegre etmek teknik tabirle "overkill" olurdu, bu site için sıfırdan bi proje yaratmak kulağa daha kolay geliyor. Buna vaktim yok ama eğer senin böyle bir proje yapmaya ilgin varsa, veya yalnızca çizgivedizi'den birşeyler indirmek istiyorsan sana ufak bir tüyo vereyim

yt-dlp yazılımı sitedeki videoları başarılı bir şekilde tespit edip indirebiliyor
```
yt-dlp 'https://www.cizgivedizi.com/film/s%C3%BCpail/super_aile'
```
![image](https://github.com/user-attachments/assets/bf56bc72-9be4-47e5-a50c-649c9001e890)

Kullanmak istersen: 
https://github.com/yt-dlp/yt-dlp

**Reply by gokhanmtlu** [reply]
Çok teşekkür ederim, inceleyeceğim.
